### PR TITLE
Add TRB Variables to Configuring Your Inventory File Topic

### PR DESCRIPTION
**Pull request by bmcelvee**
Related to https://github.com/openshift/openshift-docs/pull/11506

Note: Builds will fail until the corresponding PR is merged. If it takes longer to merge that PR, I'll pull the link from this one and add it to the other so this can be merged. 

@kalexand-rh PTAL :maple_leaf:

### CODE REVIEW

**Comment by bmcelvee** [comment]
@sdodson here's a draft PR for adding the `oreg_auth_user` and `oreg_auth_password` variables to the Configuring Your Inventory File topic.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -874,8 +874,26 @@ prefix and version from the `oreg_url` value.
 
 |`*openshift_examples_modify_imagestreams*`
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
+
+|`oreg_auth_user`
+|If `oreg_url` points to a registry requiring authentication, use the `oreg_auth_user`
```
[the remark this answers is not in the document]
Comment (sdodson) [reply install/configuring_inventory_file.adoc]: Yeah, the default for oreg_url needs to be updated to be `registry.redhat.io`.

`oreg_auth_credentials_replace` is no longer used and will generate a warning if it's set. We need to remove it from the example inventory in openshift-ansible.

[the remark this answers is not in the document]
Comment (bmcelvee) [reply install/configuring_inventory_file.adoc]: Thanks! Updates pushed

**Pull request merged**